### PR TITLE
Update Current Release & Pre-release Info

### PR DESCRIPTION
**Pull request by SteveLasker**

### CODE REVIEW

**Comment by SteveLasker** [comment]
blocked on #192

**Comment by OR13** [comment]
Merge https://github.com/veraison/go-cose/pull/193 first

File: README.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ A golang library for the [COSE specification][cose-spec]
 
 ## Project Status
 
-**Current Release**: [go-cose v1.1.0][current-release]
+**Current Supported Release**: [go-cose v1.1.0][current-release]
+
+**Current Pre-Release**: [go-cose v1.2.0-alpha.1][current-pre-release]
```
Comment (OR13) [inline README.md:13]:
```suggestion
**Current Pre-Release**: [go-cose v1.3.0-alpha.1][current-pre-release]
```

File: README.md [hunk 2]
```diff
@@ -244,3 +246,4 @@ go test -fuzz=FuzzSign1
 [mozilla-go-cose]:      http://github.com/mozilla-services/go-cose
 [veraison-go-cose]:     https://github.com/veraison/go-cose
 [current-release]:      https://github.com/veraison/go-cose/releases/tag/v1.1.0
+[current-pre-release]:  https://github.com/veraison/go-cose/releases/tag/v1.2.0-alpha.1
```
Comment (OR13) [inline README.md:249]:
```suggestion
[current-pre-release]:  https://github.com/veraison/go-cose/releases/tag/v1.3.0-alpha.1
```

**Review by OR13 — CHANGES_REQUESTED** [review]
See comments on https://github.com/veraison/go-cose/issues/190

**Review by OR13 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**